### PR TITLE
Issue #3113 - Palindrome Substring

### PR DESCRIPTION
**Pull request by EvenCode**
Add Palindome.py and palindrome_substring.py #3113 :
[https://github.com/OpenGenus/cosmos/issues/3113
](url)

Added a program that counts possible palindrome substrings within a string.

### CODE REVIEW

File: code/string_algorithms/src/palindrome_substring/palindrome_substring.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+
+/*
+ * Utilises 'iattempt' palindrome.c implementation.
+ * Part of Cosmos by OpenGenus Foundation.
+*/
+
+int isPalindromeIterative(char *input, int length)
```
Comment (arnavb) [inline code/string_algorithms/src/palindrome_substring/palindrome_substring.c]: For all of your functions, the type goes on its own line. Also, you might want to read [this](https://stackoverflow.com/q/1921539/6525260).

Comment (EvenCode) [reply code/string_algorithms/src/palindrome_substring/palindrome_substring.c]: I have made the necessary changes but still its not passing the Travis-CI.

Comment (arnavb) [reply code/string_algorithms/src/palindrome_substring/palindrome_substring.c]: @EvenCode You really don't need to worry about the Travis CI right now (It's still not fully complete). There are many files that don't pass currently. Just make sure yours does! (Check the build log to see if your file succeeded in running)

**Comment by EvenCode** [comment]
@arnavb yeah there are no errors.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**